### PR TITLE
Update style for login and signup forms.

### PR DESCRIPTION
**Pull request by mattstauffer**
Just cleaning up some little style things here and there to make it a tiny bit more pleasant. :)

Signup from this:

image

to this:

image

Login from this:

image

to this:

image

### CODE REVIEW

**Comment by adamwathan** [comment]
So purdy.

**Comment by mattstauffer** [comment]
@adamwathan is that a :+1:? :)

File: resources/views/account/create.blade.php [hunk 1]
```diff
@@ -3,40 +3,46 @@
...
+                            </div>
+                        </div>
+                        <div class="col-md-6">
+                            <div class="form-group">
+                                {{ Form::text('last_name', null, ['class' => 'form-control', 'placeholder' => 'Last name']) }}
+                            </div>
+                        </div>
+                    </div>
+
+                    {{ Form::submit('Sign up', array('class' => 'button button--primary')) }}
```
Comment (adamwathan) [inline resources/views/account/create.blade.php]: 5.4 array

File: resources/views/user/log-in.blade.php [hunk 2]
```diff
@@ -3,22 +3,24 @@
...
-            <div class="col-md-6">
-                {{ Form::open(array('action' => 'AuthController@postLogin')) }}
-                <h2>Log in</h2>
+            <div class="col-md-6 col-md-offset-3">
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <h2 class="panel-title">Log in</h2>
+                    </div>
+                    <div class="panel-body">
+                        {{ Form::open(array('action' => 'AuthController@postLogin')) }}
```
Comment (adamwathan) [inline resources/views/user/log-in.blade.php]: 5.4 array

File: resources/views/user/log-in.blade.php [hunk 3]
```diff
@@ -3,22 +3,24 @@
...
+                            {{ Form::text('email', null, ['autofocus' => 'autofocus', 'class' => 'form-control', 'placeholder' => 'Email address']) }}
+                        </div>
+                        <div class="form-group">
+                            {{ Form::password('password', ['class' => 'form-control', 'placeholder' => 'Password']) }}
+                        </div>
 
-                <div class="form-group">
-                    {{ Form::label('email', 'Email', ['class' => 'control-label']) }}<br>
-                    {{ Form::text('email', null, ['autofocus' => 'autofocus', 'class' => 'form-control']) }}
+                        {{ Form::submit('Sign in', array('class' => 'button button--primary')) }}
```
Comment (adamwathan) [inline resources/views/user/log-in.blade.php]: 5.4 array

**Comment by adamwathan** [comment]
Aside from arrays looks good. Only other thing I would say is maybe keep the labels in there with the correct `for` attribute set and hide them with the `sr-only` class so they are still there for screen readers. Not a crazy important thing to worry about but an easy accessibility win.

BootForms just lets you do this BTW ;)

`BootForm::email('email', 'Email Address')->hideLabel()`

**Comment by mattstauffer** [comment]
@adamwathan Great thinking!

And yah, definitely using BootForm in the future :)

**Pull request merged**